data/rulesets.ts

```diff
@@ -260,12 +260,11 @@ export const Rulesets: {[k: string]: FormatData} = {
 				for (const moveId of set.moves) {
 					const move = this.dex.moves.get(moveId);
 					const moveid = move.id;
-					if (hasMove[moveid]) continue;
+					if (hasMove[moveid]) return [species.baseSpecies + " has multiple copies of " + move.name + "."];
```

Comment:
```suggestion
					if (hasMove[moveid]) return [`${species.baseSpecies} has multiple copies of ${move.name}.`];
```
Is there a reason you didn't use template strings here?

Reply:
I forgot lol